lms/templates/financial-assistance/apply.html

```diff
@@ -0,0 +1,52 @@
+<%inherit file="../main.html"/>
+<%
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+%>
+
+<%namespace name="footer" file="footer.html"/>
+
+<div class="financial-assistance-wrapper">
+  <%include file="header.html"/>
+
+  <div class="financial-assistance apply-form">
+    <ol class="apply-form-list">
+      <li class="apply-form-section">
+        <h2>${_("About Me")}</h2>
+        <p>${_("The following information is part of your edX profile. All fields are required. If any information is incorrect, {link_start}edit your profile{link_end} before submitting this application.").format(
```

Comment:
Thanks for catching that. It's worth noting, though, that this is an edx-only page which is configured off by default for OpenEdX instances.